pctiler/pctiler/colormaps/noaa_c_cap.py

```diff
@@ -0,0 +1,261 @@
+from rio_tiler.types import ColorMapType
+
+
+noaa_c_map_colormap: ColorMapType = {
+    0: (0, 0, 0, 255),
+    1: (0, 0, 0, 255),
+    2: (235, 235, 235, 255),
+    3: (168, 153, 168, 255),
+    4: (143, 117, 121, 255),
+    5: (194, 204, 56, 255),
+    6: (82, 31, 0, 255),
+    7: (194, 159, 79, 255),
+    8: (235, 184, 133, 255),
+    9: (0, 235, 0, 255),
+    10: (0, 57, 0, 255),
+    11: (6, 159, 57, 255),
+    12: (108, 108, 0, 255),
+    13: (0, 94, 94, 255),
+    14: (235, 108, 0, 255),
+    15: (235, 0, 235, 255),
+    16: (57, 0, 57, 255),
+    17: (108, 0, 108, 255),
+    18: (181, 0, 181, 255),
+    19: (0, 235, 235, 255),
+    20: (235, 235, 0, 255),
+    21: (0, 0, 120, 255),
+    22: (0, 0, 236, 255),
+    23: (102, 117, 219, 255),
+    24: (1, 26, 119, 255),
+    25: (24, 24, 24, 255),
```

Comment:
I'm curious if these values beyond `25` are necessary for the data set? Looks like this is a land cover change raster and the first 24 are categories? We don't need to scale to 255 if those values aren't present in the data, and it'll make dynamic legend generation easier if they are not.

with:
![image](https://user-images.githubusercontent.com/1014341/149178661-755a92cd-e585-4a53-8306-04bc8cfc8bc2.png)

without:
![image](https://user-images.githubusercontent.com/1014341/149178621-d178a7ce-e4fc-4bfb-9f4b-68984caecaad.png)

Reply:
Related to above: if this is categorical, just confirming that the STAC collection will include the value/labels in the `item_assets.file:values` extension, similar to [io-lulc](https://planetarycomputer-staging.microsoft.com/api/stac/v1/collections/io-lulc)

Reply:
> I would prefer that, especially if we may over time be adding noaa related cmaps

Roger, will fix.

> I'm curious if these values beyond 25 are necessary for the data set? 

Shouldn't be, I just copied from the TIFF's internal map. I'll double-check manually and edit as appropriate.

>  Related to above: if this is categorical, just confirming that the STAC collection will include the value/labels in the item_assets.file:values extension, similar to io-lulc

Hm, I forgot about that (I just did `label:classes` for categories). I'll fix on the collection side, thanks for calling that out.